.github/ISSUE_TEMPLATE/new-domain-request.yml

```diff
@@ -0,0 +1,60 @@
+name: New Domain Integration
+description: Request the addition of a new domain to the Workflomics platform
+title: "[DOMAIN] Add domain: "
+labels: ["enhancement"]
+assignees: []
+
+body:
+  - type: input
+    id: domain_name
+    attributes:
+      label: Domain Name
+      description: Full name of the domain to be added
+      placeholder: e.g., proteomics-ms
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Short description of the domain and its purpose
+      placeholder: Describe what the domain includes, e.g., tools for MS-based proteomics workflows
+    validations:
+      required: true
+
+  - type: input
+    id: config_link
+    attributes:
+      label: Domain Config URL
+      description: Link to the domain's `config.json` file (must be publicly accessible)
+      placeholder: https://github.com/your-org/repo/blob/main/domains/your-domain/config.json
```

Comment:
@vedran-kasalica  shouldn't this refer to the raw.githubusercontent.com version of the url?